openapi.yaml

```diff
@@ -4680,6 +4833,10 @@ components:
         releaser:
           $ref: '#/components/schemas/DeveloperSet'
       type: object
+    Color:
+      description: RGB 24-bit color in hex.
+      pattern: '^[0-9a-fA-F]{6}$'
```

Comment:
Can we please also add `#` symbol at the begiinning? That's because on webapp we store values of colors with that symbol, so it would be a bit easier if they come in the same way from API.

Reply:
NP, but there are a few other places without `#` that can break you. Let's shoot these incompatibilities on staging. The affected endpoints are:
- `/filter/jira` - color of priorities
- `/filter/pull_requests` - color of PR labels
- `/filter/labels` - color of PR labels

Reply:
ah ok, makes sense. For the sake of consistency let's not add it then.